Update company-cadence.md

natalie-hub opened the pull request:
Updated the link to September 2019 and added in March 2020. Also removed the Day from the dates so it's only "Month YYYY" since we don't do it the same day every time. I can plan for that though, if you want to keep days in.

<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->

natalie-hub committed:
Update company-cadence.md

Updated the link to September 2019 and added in March 2020. Also removed the Day from the dates so it's only "Month YYYY" since we don't do it the same day every time. I can plan for that though, if you want to keep days in.

jasonblais commented on operations/operations/company-cadence.md:
```diff
@@ -118,8 +118,9 @@ Every 6 months we'll be asking staff to spend around 3 minutes completing a [sta
 
-* [March 13, 2019 staff enablement survey](https://docs.google.com/forms/d/1RwHyliyW6-Hh87K0L9srNB2qpDdgnu1_THcgg4lfgRs/edit#response=ACYDBNgZGE2_rGr-IPo126aHqHfDHQHdZWRjJ-vw-blT4iu6PV-FKUNf51bK7-vl9s3wll0)
-* [September 18, 2019 staff enablement survey](https://docs.google.com/forms/d/15ezikBYd4HGbhP9RiZrmCyBGjMEo_G-V8DQAKKqa6wM/edit)
+* [March 2019 staff enablement survey](https://docs.google.com/forms/d/1RwHyliyW6-Hh87K0L9srNB2qpDdgnu1_THcgg4lfgRs/edit#response=ACYDBNgZGE2_rGr-IPo126aHqHfDHQHdZWRjJ-vw-blT4iu6PV-FKUNf51bK7-vl9s3wll0)
```
@natalie-hub The first links to the survey question form, the other two to the survey results. Confirming that was intended?

natalie-hub replied:
@jasonblais - I hadn't originally updated that link since that's what was there previously. However, after your comment it does feel a bit weird to leave in there in a less digestible format. I updated the link to lead to the exact slide from the March 2020 deck that shared the historical info from March 2019.

natalie-hub committed:
Update company-cadence.md

Updated March 2019 link to results based on @jasonblais feedback on prior link.

jasonblais committed:
Correct 2019 with 2020 date
